pyramid_learning_journal/pyramid_learning_journal/scripts/initializedb.py

```diff
@@ -0,0 +1,55 @@
+import os
+import sys
+import transaction
+
+from pyramid.paster import (
+    get_appsettings,
+    setup_logging,
+)
+
+from pyramid.scripts.common import parse_vars
+
+from ..models.meta import Base
+from ..models import (
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+)
+from pyramid_learning_journal.models.mymodel import Entry
+from pyramid_learning_journal.data.data import Posts
+
+
+def usage(argv):
+    cmd = os.path.basename(argv[0])
+    print('usage: %s <config_uri> [var=value]\n'
+          '(example: "%s development.ini")' % (cmd, cmd))
+    sys.exit(1)
+
+
+def main(argv=sys.argv):
+    if len(argv) < 2:
+        usage(argv)
+    config_uri = argv[1]
+    options = parse_vars(argv[2:])
+    setup_logging(config_uri)
+    settings = get_appsettings(config_uri, options=options)
+    settings['sqlalchemy.url'] = os.environ.get('DATABASE_URL')
+
+    engine = get_engine(settings)
+    Base.metadata.drop_all(engine)
```

Comment:
Make sure that you actually do want to drop all your tables whenever you run initializedb